app/controllers/application_controller.rb

```diff
@@ -4,27 +4,21 @@ class ApplicationController < ActionController::API
   USE_SSO = Rails.configuration.sso['use_sso'].freeze
 
   def verify_sso_cookie(sf_object)
-    unless USE_SSO
-      return
-    end
+    return unless USE_SSO
 
     decrypt = OpenStax::Auth::Strategy2.decrypt(request)
     cookie_id = decrypt.dig('sub', 'salesforce_contact_id')
     contact = Contact.where(salesforce_id: cookie_id)
-    if contact.blank?
-      return_bad_request(sf_object)
-    end
+    return_bad_request(sf_object) if contact.blank?
   end
 
   def cookie_data
-    unless USE_SSO
-      return
-    end
+    return unless USE_SSO
```

Comment:
Yes, you can use it locally to do development if you don't want to spin up a local accounts install (since cookie has to be on same domain). There is a settings file here: https://github.com/openstax/openstax-salesforce-api/blob/master/config/sso.yml